include/deal.II/lac/solver_cg.h

```diff
@@ -172,6 +173,22 @@ namespace LinearAlgebra
  * run the updates on the vectors according to a variant presented in
  * Algorithm 2.2 of @cite Chronopoulos1989 (but for a preconditioner), whereas
  * the operation after the loop performs a total of 7 reductions in parallel.
+ *
+ * <h3>Preconditioned residual</h3>
+ *
+ * @p AdditionalData allows you to choose between using the explicit
+ * or implicit residual as a stopping condition for the iterative
+ * solver. This behavior can be overridden by using the flag
+ * AdditionalData::use_default_residual. A <tt>true</tt> value refers to the
+ * implicit residual, while <tt>false</tt> reverts
+ * it. The former explicitly calculates the system residual with an additional
+ * matrix-vector product, whereas the latter uses the result of the matrix-vector
+ * product already computed in other algorithm steps to derive the residual by
```

Comment:
Could you please check the text again, I have the impression that "former" and "latter" are the wrong way around. Apart from that, I'm ok with the PR and think we should merge soon. Do you have comments, @bangerth, otherwise I will merge within the next day or so (once this issue has been checked).

Reply:
Hi, yes you are right! I will change it immediately